File: python3/iniciante/1040.py
def media(a, b, c, d):
	m = (a+b+c+d) / 10
	return m

def checar(media, aprov, reprov=None):
	if (reprov==None):
		r = 'Aluno aprovado.' if media >= aprov else 'Aluno reprovado.'
		print(r)
	else:
		if (media >= aprov):
			print('Aluno aprovado.')
		elif (media < reprov):
			print('Aluno reprovado.')


n = [float(nums) for nums in input().split()]

m = media(n[0]*2, n[1]*3, n[2]*4, n[3]*1)
print('Media: {:.1f}'.format(m))

if (5 <= m <= 6.9):
	print('Aluno em exame.')

	e = float(input())
	print('Nota do exame: {:.1f}'.format(e))

	checar((m + e) / 2, 5)
	print('Media final: {:.1f}'.format((m + e) / 2))

else:
	checar(m, 7, 5)

File: c/iniciante/1040.c
#include <stdio.h>

int main() {
    double n1, n2, n3, n4, m;
    scanf("%lf", &n1);
    scanf("%lf", &n2);
    scanf("%lf", &n3);
    scanf("%lf", &n4);
    m = (n1 * 2 + n2 * 3 + n3 * 4 + n4 * 1) / 10;
    printf("Media: %.1lf\n", m);

    if (m >= 7) {
        printf("Aluno aprovado.\n");
    } else if (m < 5) {
        printf("Aluno reprovado.\n");
    } else {
        printf("Aluno em exame.\n");
        scanf("%lf", &n1);
        printf("Nota do exame: %.1lf\n", n1);
        m = (m + n1) / 2;
        if (m >= 5) {
            printf("Aluno aprovado.\n");
        } else {
            printf("Aluno reprovado.\n");
        }
        printf("Media final: %.1lf\n", m);
    }

    return 0;
}
